verify/src/main/java/com/twilio/verify/TwilioVerify.kt

```diff
@@ -128,7 +128,8 @@ interface TwilioVerify {
   )
 
   /**
-   * Deletes a **Factor** with the given **sid**
+   * Deletes a **Factor** with the given **sid**. This method calls **Verify Push API** to delete
+   * the factor and will remove the factor from local data if the API call succeeds.
```

Comment:
I think this should be `from local storage` instead of `local data`

Reply:
Done, thanks